### PR TITLE
Add auto-upgrade for build09/10/hosted-mgmt

### PR DESCRIPTION
**Pull request by bear-redhat**

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -29,6 +29,96 @@ periodics:
         requests:
           cpu: 500m
     serviceAccountName: upgrader
+- agent: kubernetes
+  cluster: build09
+  cron: 30 7 * * *
```
Comment (hongkailiu) [inline ci-operator/jobs/infra-periodics.yaml:34]: Could we exclude the weekends?

Comment (hongkailiu) [reply ci-operator/jobs/infra-periodics.yaml:34]: Yes. I would like to apply it to b01's upgrade job too.

Comment (bear-redhat) [reply ci-operator/jobs/infra-periodics.yaml]: OK. Changed.

**Pull request merged**